Polygon/eth_accounts.yaml

```diff
@@ -7,5 +7,26 @@ servers:
 paths:
   /{apiKey}:
     post:
-      $ref: ../ethereum/eth_accounts.yaml#/components/pathItems/path/post
```

Comment:
@opman129 we don't want to remove the entire ref here, we only want to update the description part